### PR TITLE
Mpgddirc geo fix

### PR DESCRIPTION
**Pull request by mposik1983**
### Briefly, what does this PR introduce?
Fixes Bad volume warnings listed in Issue #234 
https://github.com/eic/epic/issues/234 

### What kind of change does this PR introduce?
- [x] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [x] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No, it should not
### Does this PR change default behavior?
No

### CODE REVIEW

File: compact/tracker.xml [hunk 1]
```diff
@@ -158,13 +158,14 @@
...
-  <include ref="tracker/mpgd_dirc.xml"/>
-  <include ref="tracker/silicon_disks.xml"/>
-  <include ref="tracker/support_service_assembly.xml"/>
-  <include ref="tof_barrel.xml"/>
-  <include ref="tof_endcap.xml"/>
+	    <include ref="tracker/vertex_barrel.xml"/>
+	    <include ref="tracker/silicon_barrel.xml"/>
+	    <include ref="tracker/mpgd_barrel.xml"/>
+            <include ref="tracker/mpgd_dirc.xml"/>
+            <include ref="dirc.xml"/>
```
Comment (wdconinc) [inline compact/tracker.xml:165]: Isn't this going to load the DIRC twice?

File: compact/tracker.xml [hunk 2]
```diff
@@ -158,13 +158,14 @@
...
+	    <include ref="tracker/silicon_disks.xml"/>
+	    <include ref="tracker/support_service_assembly.xml"/>
+	    <include ref="tof_barrel.xml"/>
```
Comment (wdconinc) [inline compact/tracker.xml:168]: Same with TOF.

File: compact/tracker.xml (hunk 1 above)
Comment (mposik1983) [reply compact/tracker.xml:165]: Yes, I had it there to see the dirc with the tracking system. I forgot to remove it before I pushed

File: compact/tracker.xml (hunk 2 above)
Comment (wdconinc) [reply compact/tracker.xml:167]: Nevermind this was in tracker already

**Comment by mposik1983** [comment]
@wdconinc is this pr failing the epic_brycecanyon benchmarks because the mpgd_dirc is not supposed to be in that configuration (the one with the silicon layer in the barrel calorimeter)?

**Comment by wdconinc** [comment]
I don't think that silicon layer is implemented, and it wasn't on Silvia's latest presentation. I think both arches and brycecanyon are using the DIRC MPGD detector.

**Comment by wdconinc** [comment]
This is failing because brycecanyon doesn't have mrich, and until yesterday we had the mrich in brycecanyon by mistake.

**Comment by wdconinc** [comment]
I'll fix the physics benchmarks to do this correctly.

**Comment by mposik1983** [comment]
Ok, thanks for the clarification. I didn't realize the dirc mpgd was going to be in both configurations. Good to know :)

**Comment by wdconinc** [comment]
This is what I have from Silvia's presentation. Also the detector parameter table has the DIRC MPGD in the baseline and no alternative.

image

**Comment by sly2j** [comment]
That is a mistake. BryceCanyone does NOT have the MPGD behind the DIRC. The imaging ECAL first layer serves the same role.

**Comment by sly2j** [comment]
Screen Shot 2022-10-20 at 12 08 04 PM
FYI, this is the spreadsheet from GD&I. It is important we do this right to have a meaningful comparison.

**Review by wdconinc — APPROVED** [review]
Looks good. Thanks for fixing (I assume it's fixed and no more warnings).

**Pull request merged**